sdk/anomalydetector/Azure.AI.AnomalyDetector/README.md

```diff
@@ -61,6 +61,22 @@ string endpoint = "<endpoint>";
 var client = new AnomalyDetectorClient(new Uri(endpoint), new DefaultAzureCredential());
 ```
 
+## Key concepts
+
+### Thread safety
+We guarantee that all client instance methods are thread-safe and independent of each other ([guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-methods-thread-safety)). This ensures that the recommendation of reusing client instances is always safe, even across threads.
+
+### Additional concepts
+<!-- CLIENT COMMON BAR -->
+[Client options](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md#configuring-service-clients-using-clientoptions) |
```

Comment:
For my two cents, I think there's value in the locality of information.  The more we're able to present at-a-glance for casual readers, the better we're able to communicate.   Every jump that we ask developers to make reduces the likelihood that readers will actually follow them and consume the information.    

I don't disagree with the underlying concern about the need to keep multiple copies of the information up-to-date and in-sync;  perhaps this would be a good area to consider a tool that performs maintenance automatically to reduce friction and the chance of manual errors.

Reply:
I don't have concerns with the general section in each place I was mostly reacting to the Additional concepts list which is just a bunch of other links with anchors on them that will be easy to break and if we change any of those headings for example all these links across all these files will have to be updated.

Reply:
Does the link checker check bookmarks? If not, maybe we should update it to prevent this situation.

Reply:
The link checker doesn't have a great way to validate anchor/bookmarks, so it currently doesn't. However having those links checked is only a solution to finding breaks it isn't a solution to the larger problem of having to potentially update every README file in the repo if any of those change, that was my only point. If you guys think it is better to have a list of links in every file instead of just one that links to more content and potentially this list of links then that is up to you.

Reply:
@weshaggard I agree with you -- those links can break if we change any of the 
If we have a way to at least detect when that break happens that will be good enough for now -- I am considering having a script of some other mechanism of updating those links at some point.

For now I think we should add the links and we can reconsider this the first time a link breaks.